demo/src/com/google/maps/android/utils/demo/GeoJsonDemoActivity.java

```diff
@@ -0,0 +1,51 @@
+package com.google.maps.android.utils.demo;
+
+import com.google.android.gms.maps.model.LatLng;
+import com.google.maps.android.geojson.GeoJsonFeature;
+import com.google.maps.android.geojson.GeoJsonLayer;
+import com.google.maps.android.geojson.GeoJsonPoint;
+import com.google.maps.android.geojson.GeoJsonPolygonStyle;
+
+import org.json.JSONException;
+
+import android.graphics.Color;
+import android.util.Log;
+
+import java.io.IOException;
+import java.util.HashMap;
+
+public class GeoJsonDemoActivity extends BaseDemoActivity {
+
+    private final static String mLogTag = "GeoJsonDemo";
+
+    protected int getLayoutId() {
+        return R.layout.geojson_demo;
+    }
+
+    @Override
+    protected void startDemo()  {
```

Comment:
Can you zoom to the data you've added in here? It looks broken unless you know what you're looking for.